### PR TITLE
Move to 4.x

### PR DESCRIPTION
**Pull request by blond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,14 +102,11 @@ module.exports = function(config) {
...
 ---------------------
 
-По БЭМ-методологии шаблоны к каждому блоку хранятся в отдельных файлах с расширением `.bemhtml.js` или `.bemtree.js` в директориях блоков. Чтобы использовать шаблоны, необходимо собрать их исходные файлы.
+По БЭМ-методологии шаблоны к каждому блоку хранятся в отдельных файлах с расширением `.bemhtml.js` в директориях блоков. Чтобы использовать шаблоны, необходимо собрать их исходные файлы.
 
-Отдельные файлы с шаблонами (`.bemhtml.js` или `.bemtree.js`) собираются в один общий файл (`?.bemhtml.js` или `?.bemtree.js`) с помощью одной из следующих технологий:
-
-* [bemhtml](api.ru.md#bemhtml)
-* [bemtree](api.ru.md#bemtree)
+Отдельные файлы с шаблонами собираются в один общий файл (`?.bemhtml.js`) с помощью [bemhtml](api.ru.md#bemhtml) технологии.
```
Comment (tadatuta) [inline README.md]: с помощью технологии [bemhtml](api.ru.md#bemhtml).

File: techs/bemhtml.js [hunk 2]
```diff
@@ -5,20 +5,20 @@ var bundle = require('../lib/bundle');
...
- * @param {Boolean}   [options.devMode=false]            Sets `devMode` option for convenient debugging. If `devMode` is
- * set to true, code of templates will not be compiled but only wrapped for development purposes.
- * @param {Object}    [options.requires]                 Names of dependencies which should be available from
- * code of templates.
+ * @param {Object}    [options]                        Options
+ * @param {String}    [options.target='?.bemhtml.js']  Path to a target with compiled file.
+ * @param {String}    [options.filesTarget='?.files']  Path to a target with FileList.
+ * @param {String[]}  [options.sourceSuffixes]         Files with specified suffixes involved in the assembly.
+ * @param {String}    [options.exportName='BEMHTML']   Name of BEMHTML template variable.
+ * @param {Boolean}   [options.devMode=false]          Sets `devMode` option for convenient debugging. If `devMode` is
```
Comment (tadatuta) [inline techs/bemhtml.js]: `devMode` же всё?

**Comment by tadatuta** [comment]
please provide support for `naming` option (refer to https://github.com/tadatuta/enb-bemxjst/commit/75e45bd6adb76ad834aadd212e6da805bf5592dc).

**Comment by tadatuta** [comment]
I think an option to automatically remove base templates should be provided to support current versions of base libraries (refer to https://github.com/tadatuta/enb-bemxjst/commit/db7bff1e47d2b106155033a0c9ab0fdb511e7a03).

**Comment by blond** [comment]
:up:

**Comment by blond** [comment]
:up:

**Pull request merged**